src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -183,7 +182,10 @@ export const createVideojsPlayer = (
     isInitialized = true;
     // setTimer
     if (hasAttendance) {
-      interval = player.setInterval(trackAttendance, PUSH_ATTENDANCE_DELAY);
+      interval = player.setInterval(
+        trackAttendance,
+        appData.attendanceDelay * 1000,
```

Comment:
IMO this should not be done here but in the django view. In the commit message you explain delegate this value to the backend application. The front should not be responsible to modify it then.

Reply:
Are you talking about the fact is *1000 in the front?